### PR TITLE
@core-util/watcher tutorial

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: core-util-watcher/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    console.log(`L1ToL2 hash: ${l1ToL2_Hash}`)
+    const l1ToL2_L2Receipt = await watcher.getL2TransactionReceipt(l1ToL2_Hash)
+    console.log(`L1ToL2 L2 TX hash: ${l1ToL2_L2Receipt.transactionHash}`)
+}
+
+
+
+
+
+
```
Comment (elenadimitrova) [inline core-util-watcher/index.js:52]: Remove extra space here.

File: core-util-watcher/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Tracing Multilayer Transactions
+
+[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord.com/channels/667044843901681675)
+[![Twitter Follow](https://img.shields.io/twitter/follow/optimismPBC.svg?label=optimismPBC&style=social)](https://twitter.com/optimismPBC)
+
+Optimistic Ethereum lets you send messages between L1 and L2, in either direction.
+L1 to L2 messages, for example, can be "this address locked asset X in the bridge
```
Comment (elenadimitrova) [inline core-util-watcher/README.md:7]: This explanatory wording isn't ideal. Can we reword to something like
"Optimistic Ethereum lets you send messages between L1 and L2, in either direction. There are always two transactions involved - one on each layer. 
For example when sending a token balance from L1 to L2 there is a transaction which locks the token balance in the bridge on L1 and a second which mints the equivalent token amount on L2. For withdrawals the opposite occurs - again in two steps - the token amount is burned on L2, then the equivalent amount (previously locked) is transferred out of the bridge to the user on L1.

File: core-util-watcher/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    l2: {
+      provider: new JsonRpcProvider(`https://kovan.optimism.io`),
+      messengerAddress: '0x4200000000000000000000000000000000000007'
+    }
+  })
+
+
+  const mainnetWatcher = new Watcher({
+    l1: {
+      provider: new JsonRpcProvider(`https://mainnnet.infura.io/${infuraID}`),
```
Comment (elenadimitrova) [inline core-util-watcher/index.js:27]: typo mainnnet -> mainnet

File: core-util-watcher/index.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      // On Kovan, the value for mainnet is
```
Comment (elenadimitrova) [inline core-util-watcher/index.js:28]: What do you mean by this comment "On Kovan, the value for mainnet is" ?

File: core-util-watcher/index.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      messengerAddress: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1'
+    },
+    l2: {
+      provider: new JsonRpcProvider(`https://mainnet.optimism.io`),
+      messengerAddress: '0x4200000000000000000000000000000000000007'
+    }
+  })  
+
+
+const watcher = kovanWatcher;   // mainnetWatcher
```
Comment (elenadimitrova) [inline core-util-watcher/index.js:38]: I tried running the example for a mainnet transaction `0x122abee5d2f41da16a40e7edd0c5460d8f50683a46e1765fee2a8a8eb8b01cc2` but it throws with 
```
(node:33028) UnhandledPromiseRejectionWarning: Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.4.3)
```

File: core-util-watcher/README.md (hunk 2 above)
Comment (qbzzt) [reply core-util-watcher/README.md]: Thank you, you're right - changed.

File: core-util-watcher/index.js (hunk 4 above)
Comment (qbzzt) [reply core-util-watcher/index.js:28]: That I forgot to remove the comment :-(

Comment (qbzzt) [reply core-util-watcher/index.js:28]: I should have deleted that comment.

File: core-util-watcher/index.js (hunk 5 above)
Comment (qbzzt) [reply core-util-watcher/index.js]: Fixed, thank you.

Comment (qbzzt) [reply core-util-watcher/index.js]: Solved it, my bad.

File: core-util-watcher/index [hunk 6]
```diff
@@ -0,0 +1,45 @@
+L2ToL1 L2 TX hash: 0xa018b12df969c0f65aec3c3d08c94332525a309cdec38206eef74319c1d1dd30
```
Comment (elenadimitrova) [inline core-util-watcher/index:1]: Pls remove this file, I think it was committed by mistake

**Review by elenadimitrova — APPROVED** [review]

**Pull request merged**